book/src/ci/github.md

```diff
@@ -157,6 +157,51 @@ By default cargo-dist breaks build tasks onto more machines than strictly necess
 
 
 
+### Checking what your build linked against
+
+> since 0.4.0
+
+Although most Rust builds are statically linked and contain their own Rust dependencies, some crates will end up dynamically linking against system libraries. It's useful to know what your software picked up&mdash;sometimes this will help you catch things you may not have intended, like dynamically linking to OpenSSL, or allow you to check for package manager-provided libraries your users will need to have installed in order to be able to run your software.
```

Comment:
emdashes want whitespace around them, right?

Reply:
Depends on our style guide, I guess. I usually don't, but happy to add spaces.